versioned/spi/src/main/java/org/projectnessie/versioned/store/DefaultStoreWorker.java

```diff
@@ -44,47 +42,63 @@ public static StoreWorker instance() {
     return Lazy.INSTANCE;
   }
 
+  public static byte payloadForContent(Content c) {
```

Comment:
May be move the serializer registry stuff into a new top-level class to avoid confusion with "worker" names?

Reply:
Hm, it's tightly coupled and I'm not so much a fan of opening that `Registry` class